src/content/api/cli.md

```diff
@@ -293,21 +293,21 @@ You can also pass multiple values:
 npx webpack --config-name first --config-name second
 ```
 
-__Merge two or more different webpack configurations__
+**Merge two or more different webpack configurations**
 
-You can merge two or more different webpack configurations with the help of `--merge`: 
+You can merge two or more different webpack configurations with the help of `--merge`:
 
 ```bash
 npx webpack --config ./first.js --config ./second.js --merge
 ```
 
-__Print result of webpack as a JSON__
+**Print result of webpack as a JSON**
```

Comment:
This was done by prettier?

Reply:
It doesnt change anything visually.

Reply:
It is. And there's no option to force it to use `__`, that's why I've made changes to `.markdownlint.json`.